metadata_service/api/data_quality_checks.py

```diff
@@ -0,0 +1,26 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+from http import HTTPStatus
+from typing import Any, Tuple
+
+from amundsen_common.models.data_quality_checks import TableQualityCheckSchema
+from flasgger import swag_from
+from flask_restful import Resource
+
+from metadata_service.proxy import get_proxy_client
+
+
+class TableQualityChecksAPI(Resource):
+    def __init__(self) -> None:
+        self.client = get_proxy_client()
+        super(TableQualityChecksAPI, self).__init__()
+
+    @swag_from('swagger_doc/data_quality_checks/table_quality_checks_get.yml')
+    def get(self, table_uri: str) -> Tuple[Any, HTTPStatus]:
```

Comment:
would make sense to provide (in time) put/post api so external systems can dynamically register new checks